h5t.go

```diff
@@ -188,7 +188,7 @@ func (t *Datatype) Size() uint {
 }
 
 // SetSize sets the total size of a Datatype.
-func (t *Datatype) SetSize(sz uint) error {
+func (t *Datatype) SetSize(sz int) error {
```

Comment:
Actually, on a 32 bit arch, `int` and `size_t` are probably still the same size (though not guaranteed). I think we may want to have some init checks for things like this at some stage.